main.js

```diff
@@ -16,3 +17,137 @@ listItems.forEach((element) => {
     cover.classList.remove('d-flex');
   });
 });
+
+const projectCards = [
+  {
+    title: 'Keeping track of hundreds of components',
+    image: './IMG/modal-popup.png',
+    technologies: ['Ruby on rails', 'CSS', 'JavaScript', 'Bootstrap', 'CodePen', 'Terminal', 'CodeKit', 'GitHub'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dumm. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dumm',
+    button: 'See Project',
+    links: ['https://jeffkay16.github.io/Portfolio-Website/', 'https://github.com/JEFFKAY16/Portfolio-Website.git'],
+  },
+];
+const [project] = projectCards;
+const popupSection = document.querySelector('.popup-section');
+popupSection.innerHTML = ` 
+<div class="popup-card">
+    <i class="fa fa-times popup-close-btn" aria-hidden="true"></i>
+    <div class="popup-img">
+      <img class="img-block" src="${project.image}" alt="#">
+    </div>
+    <div class="bottom-card-popup">
+      <h4 class="popup-title">${project.title}</h4>
+      <ul class="popup-tech" id="foo">
+        <li class="mobile-only">${project.technologies[0]}</li>
+        <li class="mobile-only">${project.technologies[1]}</li>
+        <li class="mobile-only">${project.technologies[2]}</li>
+        <li class="desktop-only">${project.technologies[3]}</li>
+        <li class="desktop-only">${project.technologies[4]}</li>
+        <li class="desktop-only">${project.technologies[5]}</li>
+        <li class="desktop-only">${project.technologies[6]}</li>
+        <li class="desktop-only">${project.technologies[7]}</li>
+      </ul>
+      <p class="popup-text">${project.description}</p>
+      <div class="popup-btns">
+        <a href="${project.links[0]}" class="popup-btn font-f see-live">See Live <span><i class="fa fa-external-link"></i></span></a>
+        <a href="${project.links[1]}" class="popup-btn font-f see-source">See Source <span><i class="fa fa-github"></i></span></a>
+      </div>
+    </div>
+</div>`;
+
+const pop = () => {
+  popupSection.classList.remove('d-none');
+};
+
+const popupClose = document.querySelector('.popup-close-btn');
+popButton.forEach((element) => {
+  element.addEventListener('click', () => {
+    pop();
+  });
+});
```

Comment:
- [ ] The `X` button is not there I can't close the pop-up.

Reply:
Hi, @dasileker, 

Thank you for your review. However, the issue you referenced works fine for both the previous reviewer and my coding partners' machine (whose code was accepted). 

I have this as my ultimate chance and I  am afraid that it will cost me. It's difficult to debug what I am not experiencing and I am assuming this issue might be due to your internet connection not loading all the font resources.

Kindly see the popup previews below for your reference.

![Screenshot 2021-10-08 at 1 00 53 PM](https://user-images.githubusercontent.com/84200026/136547863-e2dfcb3f-314d-4a9a-8a9d-7eef0d0a6eb5.png)
![Screenshot 2021-10-08 at 1 01 08 PM](https://user-images.githubusercontent.com/84200026/136547884-2844ebe4-db8d-4568-8519-d8adc68dd939.png)
![Screenshot 2021-10-08 at 1 03 41 PM](https://user-images.githubusercontent.com/84200026/136547890-248fc4b7-391a-4710-9071-fe63a81cfffa.png)